debian/rules

```diff
@@ -150,6 +150,8 @@ override_dh_python2:
 	dh_python2 -p ceph-common
 	dh_python2 -p ceph-base
 	dh_python2 -p ceph-osd
+	dh_python2 -p ceph-mon
+	dh_python2 -p ceph-mgr
 
 override_dh_python3:
 	for binding in rados cephfs rbd; do                                \
```

Comment:
@smithfarm could you add `dh_python3 -p ceph-mgr` to `override_dh_python3` accordingly? and maybe we don't need to run ceph-mon through dh_python2?